packages/manager-react-components/src/hooks/date/useFormattedDate.spec.tsx

```diff
@@ -0,0 +1,67 @@
+import { renderHook } from '@testing-library/react';
+import {
+  useFormattedDate,
+  defaultUnknownDateLabel,
+  DateFormat,
+} from './useFormattedDate';
+
+jest.mock('react-i18next', () => ({
+  useTranslation: () => {
+    return {
+      t: (key: string) => key,
+      i18n: {
+        language: 'fr-FR',
+      },
+    };
+  },
+}));
```

Comment:
May be relevant to do test in english too ?

Reply:
I did them in a different file since we can only mock a module once and I can't change the value dynamically